content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,206 @@
+   possible to get back to the :guilabel:`Test` and :guilabel:`Demo`. Equally is not possible to go
+   from :guilabel:`Test` to :guilabel:`Official`.
 
-.. image:: italy/journal.png
-   :align: center
-   :alt: FatturaPA feature in the sales journal's settings
+After having selected the right mode, you need to accept the terms and conditions right below the
+mode selection. Once you have :guilabel:`Saved` the mode you want, you can start recording your
+Accounting transactions in Odoo.
 
 Issue invoices
---------------
+==============
 
-Send an invoice the way you would do it normally. The EDI process proceeds automatically.
+The **FatturaPA** feature is automatically active, you can check it in
+:menuselection:`Configuration --> Journals --> Customer Invoices -->  Advanced Settings`.
+You can create a new invoice by going to :menuselection:`Accounting dashboard --> New invoice`, once
+confirmed the e-invoice will be generated and sent.
 
-.. image:: italy/sending.png
-   :align: center
-   :alt: Asynchronous sending on a customer invoice
+.. warning::
+   The e-invoice is only sent to the government if you are in :guilabel:`Official` mode.
 
-You can check the current status of your customer invoice under the **Electronic invoicing** field.
+You can check the current status of your customer invoice under the :guilabel:`Electronic invoicing`
+field. The xml file is accessible in the chatter of the invoice.
 
 .. image:: italy/processing.png
    :align: center
    :alt: Electronic invoicing status (waiting for confirmation)
 
+Vendor bills
+============
+
+As for invoices, when creating a vendor bill, the :guilabel:`FatturaPA` option is automatically
+activated in the Vendor bill Journals' advanced settings.
+When inserting taxes in a vendor bill, you have the possibility to choose **Reverse Charge** taxes,
+these are automatically activated in the Italian fiscal position. By going to
+:menuselection:`Configuration --> Taxes`, you will see that the 10% and 22% services and goods taxes
+are activated and have the correct tax grids; these are set up automatically to ensure the correct
+booking of accounting entries and display of the tax report.
+Once the bill is confirmed, a banner appears, and the bill can be sent to the EDI service.
+
+Taxes
+=====
+
+Specific configurations are required for **Reverse Charge** type of taxes. In such case, the seller
+does not charge the client for VAT but the buyer is paying for it. There are two main cases,
+Internal and External Reverse Charge.
+
+External Reverse Charge
+-----------------------
+
+Vendors of Italian companies selling and buying goods from other EU countries or services from
+non-EU countries are responsible for collecting VAT from the buyer and giving it back to the State.
+When the buyer is from another State, he is the responsible for collecting the right VAT amount and
+sending it to the Tax Agency.
+
+Invoices
+--------
+
```

Comment:
Where are we? Where to perform the below steps? Maybe a screenshot would be helpful here?